Aptoide/bds-sdk
# [APPC-1691] Check for purchases made by guest wallet

**What does this PR do?**

Checks for purchases made by the guest wallet, with or without wallet installed
Consumes unconsumed purchases made by the guest wallet, with or without wallet installed

**Database changed?**

   Yes | No

**Where should the reviewer start?**

AppCoinsStubHelper.java
AppCoinsBillingWrapper.java

**How should this be manually tested?**

1- On toolbox, press the query purchases without wallet and check if it returns an unconsumed purchase (you have to use debug, since toolbox is unable to verify the purchase)
2- Install the AppCoinsWallet
3- Do step 1 again
4- Do a purchase with the Wallet
5- Do step 1 again, and check if the list has 2 items

**What are the relevant tickets?**

https://aptoide.atlassian.net/browse/APPC-1691


**Code Review Checklist**
